### DISCUSSION TITLE
Ability to specify additional ports in pod overrides

### DISCUSSION THREAD

**Discussion by TehNeon** [jeremylvln/Shulker#482] [Ideas]
It'd be nice to have the ability to specify additional ports on pods. My usage case for this is that I want to use UnifiedMetrics (https://github.com/Cubxity/UnifiedMetrics) for server metrics. I currently can do a workaround with this by using the configuring the plugin to utilize Prometheus push gateway, but I'd prefer to utilize scraping with a ServiceMonitor/PodMonitor. I'd also like to be able to utilize the GeyserMC (https://github.com/GeyserMC/Geyser) plugin for my proxy platform of choice (Velocity) to allow Bedrock players to connect.

**Comment by jeremylvln** [comment]
Totally a thing to do. Will do shortly!

**Reply by jroberts235** [reply]
I'm looking forward to this feature.  I'd like to run the GeyserMC plugin as well.
